packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4734,11 +4734,60 @@ class Listener extends SingleChildRenderObjectWidget {
 ///
 /// This widget creates a separate display list for its child, which
 /// can improve performance if the subtree repaints at different times than
-/// the surrounding parts of the tree. Specifically, when the child does not
-/// repaint but its parent does, we can re-use the display list we recorded
-/// previously. Similarly, when the child repaints but the surround tree does
-/// not, we can re-record its display list without re-recording the display list
-/// for the surround tree.
+/// the surrounding parts of the tree.
+///
+/// This is useful since [RenderObject.paint] may be triggered even if its
+/// associated [Widget] instances' did not change or rebuild. A [RenderObject]
```

Comment:
not sure what "instances'" here is referring to. Is there a missing noun? Or is the apostrophe extraneous?

Reply:
Ah yes, an extra apostrophe